src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -544,6 +544,15 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     void TermControl::_MouseClickHandler(Windows::Foundation::IInspectable const& /*sender*/,
                                          Input::PointerRoutedEventArgs const& args)
     {
+        // Ignore mouse events while the terminal does not have focus. 
+        // This prevents the user from selecting and copying text if they 
+        // click inside the current tab to refocus the terminal window. 
+        if (!_focused) 
+        {
+            args.Handled(true);
+            return;
+        }

```

Comment:
Just noticed that in CMD/Powershell (outside of WT), an unfocused window can still be scrolled w/ touch. Since we're exiting before line 604, I suspect we won't be able to do that here. Could you double check and fix if that's the case?

Reply:
It took me a bit to get the touch simulation working, but I have confirmed that this is the case. A touch and drag will scroll the original CMD and PowerShell terminals without focusing them.

With the focus check at the top of the MouseClick event handler, a touch and drag does not scroll and the window is incorrectly focused. 
Moving the focus check inside the PointerDeviceType() == Mouse check does enable scrolling, but the window is still focused. 

It looks like a slightly more complicated solution will be necessary. I'll start looking into it tonight.

Reply:
I started digging into the issue touch focus/scrolling issue, but I will not have time to properly fix it. I will be out of town for a week starting tomorrow and will be unable to commit any time to this PR. 

I did confirm that the touch and drag issue was present before the changes of this PR, and have made a small change that returns the behavior of a touch and drag event to how it originally was while still fixing #670 (in regards to mouse click events).

I'm more than happy to open a new issue for the touch and drag problem. When I return I will be able to devote the time to resolve it. In the mean time, I would hate to see this PR remain open and stagnant for a week because of an already present bug. However it is not my repo, so I'll defer to your preferences.

Reply:
Don't worry about it. :) Thanks for looking into it. I'll put an issue in (if there isn't one already) and let's get the PR in!

Reply:
So, I have a Surface Pro and I've been comparing what your PR has with an admin Powershell instance. I think they're both acting the same way with regards to providing focus. I don't think we'll need an issue for this, then.